migrations/V144__Create_Intake_Request_State_and_Step.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TYPE system_intake_step AS ENUM (
+  'INITIAL_REQUEST_FORM',
+  'DRAFT_BUSINESS_CASE',
+  'GRT_MEETING',
+  'GRB_MEETING',
+  'FINAL_BUSINESS_CASE',
+  'DECISION_AND_NEXT_STEPS'
+);
+
+CREATE TYPE system_intake_state AS ENUM (
```

Comment:
This is pretty minor, but I think that the column/field for storing Open/Closed should maybe have a different name than "state"? We've been using "state" as a term to refer to _all_ of the data we persist about an intake's state in the workflow (current step, open/closed, task states); using it as the name for this specific column/field as well is a little confusing.

That said, I'm not sure off the top of my head what a better name would be; we _could_ do a boolean `isOpen` column, but that doesn't convey the meaning as clearly. `decision_state`/`DecisionState` could get confused with the state of the decision step. I'm definitely open to suggestions, and if we can't come up with a good replacement, I'm fine going ahead with "state".

Reply:
@DylanSpOddball, Lee and I had a similar discussion about this originally. Lee pointed out that `state` fits the paradigm if we are talking about task list states as well. It also give us flexibility if there is somehow another state added in the future (`deleted` or something). Otherwise, I was also thinking about a boolean column. I'm fine with state here, but would be fine with a change as well.

Reply:
My (small) issue is that whether an intake is open or closed is definitely _part_ of an intake's state, but it's not the _entirety_ of the state. Using "state" as the name for just open/closed might get confusing when we talk about how statuses are calculated based on an intake's state; it might sound like we're just referring to this column, when in fact statuses are based on currentStep + open/closed + each step's state.

Reply:
Gotcha; without a clear alternative (which I can't think of either), going ahead with it is fine, and wasn't worth holding this PR up for. Good call on moving forward with it.